sdk/tests/integration/TransactionReceiptQueryIntegrationTest.cc

```diff
@@ -83,25 +88,127 @@ class TransactionReceiptQueryIntegrationTest : public ::testing::Test
+
+  // Then
+  EXPECT_EQ(txReceipt.getStatus(), Status::SUCCESS);
+  EXPECT_TRUE(txReceipt.getAccountId().has_value());
+  EXPECT_TRUE(txReceipt.getExchangeRates().has_value());
+  EXPECT_TRUE(txReceipt.getExchangeRates().value().getCurrentExchangeRate().has_value());
+}
+
+//-----
+TEST_F(TransactionReceiptQueryIntegrationTest, ExecuteEmptyAccountCreateTransaction)
+{
+  // Given / When
+  TransactionReceipt txReceipt;
+  EXPECT_THROW(txReceipt = AccountCreateTransaction().execute(getTestClient()).getReceipt(getTestClient()),
+               Hedera::PrecheckStatusException);
+
+  // Then
+  EXPECT_EQ(txReceipt.getStatus(), Status::UNKNOWN);
+}
+
+//-----
+TEST_F(TransactionReceiptQueryIntegrationTest, ExecuteAccountCreateTransactionFromProtobuf)
 {
   // Given
   const auto testPublicKey = ED25519PrivateKey::generatePrivateKey()->getPublicKey();
 
-  AccountCreateTransaction testAccountCreateTransaction;
-  testAccountCreateTransaction.setKey(testPublicKey);
+  auto body = std::make_unique<proto::CryptoCreateTransactionBody>();
+  body->set_allocated_key(testPublicKey->toProtobuf().release());
+  body->set_initialbalance(static_cast<uint64_t>(Hbar(10ULL).toTinybars()));
+  body->set_receiversigrequired(false);
+  body->set_memo("Test Account me Memo");
+  body->set_max_automatic_token_associations(static_cast<int32_t>(3U));
+  body->set_allocated_staked_account_id(getTestAccountId().toProtobuf().release());
+  body->set_decline_reward(true);
 
-  const TransactionResponse txResponse = testAccountCreateTransaction.execute(getTestClient());
+  proto::TransactionBody txBody;
+  txBody.set_allocated_cryptocreateaccount(body.release());
 
   // When
-  const TransactionReceipt txReceipt = txResponse.getReceipt(getTestClient());
+  TransactionReceipt txReceipt;
+  EXPECT_NO_THROW(txReceipt = AccountCreateTransaction(txBody).execute(getTestClient()).getReceipt(getTestClient()));
 
   // Then
   EXPECT_EQ(txReceipt.getStatus(), Status::SUCCESS);
   EXPECT_TRUE(txReceipt.getAccountId().has_value());
   EXPECT_TRUE(txReceipt.getExchangeRates().has_value());
   EXPECT_TRUE(txReceipt.getExchangeRates().value().getCurrentExchangeRate().has_value());
+}
+
+//-----
+TEST_F(TransactionReceiptQueryIntegrationTest, ExecuteFileCreateTransaction)
+{
+  // Given
+  const std::vector<std::byte> byteCode = internal::HexConverter::hexToBytes(
+    json::parse(std::ifstream(std::filesystem::current_path() / "hello_world.json", std::ios::in))["object"]
+      .get<std::string>());
+
+  const std::unique_ptr<PrivateKey> operatorPrivateKey = ED25519PrivateKey::fromString(
+    "302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137");
+  const std::shared_ptr<PublicKey> operatorPublicKey = operatorPrivateKey->getPublicKey();
+
+  // When
+  TransactionReceipt txReceipt;
+  ASSERT_NO_THROW(txReceipt = FileCreateTransaction().execute(getTestClient()).getReceipt(getTestClient()));
```

Comment:
```suggestion
  EXPECT_NO_THROW(txReceipt = FileCreateTransaction().execute(getTestClient()).getReceipt(getTestClient()));
```

This is how I've been choosing between using `EXPECT` and `ASSERT`:
 - `EXPECT`: when the functionality within the macro encompasses the purpose of the test. For this test, the purpose is to get a `TransactionReceipt` for a `FileCreateTransaction` and make sure the proper fields are populated. Since getting the receipt is wrapped in this macro here, I believe it should `EXPECT_NO_THROW`.
 - `ASSERT`: when the functionality within the macro is necessary for a future "test" or "check" to work. This can mean two things:
  1) A Hedera transaction/query depends on the completion of another transaction/query. For example, if testing the creation of a smart contract, this would require the smart contract bytecode to exist in a file first. In order to create that file, you would need to use a `FileCreateTransaction`. This file creation, I think, should be wrapped in an `ASSERT_NO_THROW`, as the test should not even bother trying to create the smart contract if the file cannot be created; the test should just show failed and the problem can be addressed.
  2) If you will be indexing into an object and you want to make sure the values exist. This can be, for example, if you need to check to values in a `std::vector`, you `ASSERT` the size of the vector is equal to the amount of values you need to check. (e.g. if you need to check two values: `ASSERT_EQ(vector.size(), 2); EXPECT_EQ(vector.at(0), ...);` OR, if you need to make sure a value is contained in a `std::optional` object. (e.g. `ASSERT_TRUE(txReceipt.getAccountId().has_value()); EXPECT_EQ(txReceipt.getAccountId().value(), ...);`

I'm open to discussion on these if you have different viewpoints, but I think this help distinguishes in test what it is for which we're actually testing, versus just test set-up and assumptions.

Reply:
Fixed my latest changes.